components/src/main/res/layout/component_editable_list_view.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
It doesn't look like we are making use of any of the constraint layout features in this layout? If we don't need it we can keep it simple and just use a LinearLayout.